### PR TITLE
PCHR-2991: Include shoreditch css files in onboarding form page

### PR DESCRIPTION
**Pull request by bmnepali**
## Overview
This PR includes the `org.civicrm.shoreditch` css file (custom-civicrm.js) file in onboarding form page as it contains the datepicker styles.

## Technical Details
1. `css` file(s) from `org.civicrm.shoreditch`  extension is added only for onboarding-form page in file civihr_employee_portal/civihr_employee_portal.module.
```php
/**
 * Implements hook_css_alter().
 */
function civihr_employee_portal_css_alter(&$css) {
  ...
  if (_isCiviCRM()) {
    ...
  } else {
    if (!strpos(url(current_path(), array('absolute' => TRUE)), 'onboarding-form')) {
      _remove_shoreditch_resources($css);
    }
  }
}
```

## Comments
- The styles from `org.civicrm.shoreditch` were excluded  all pages except civicrm pages.

## Tests
⏹ PHP Unit Tests - not added

### CODE REVIEW

**Comment by mickadoo** [comment]
In description:

- `s/The styles form/The styles from/`
- css file(s) from ``

File: civihr_employee_portal/civihr_employee_portal.module [hunk 1]
```diff
@@ -32,7 +32,9 @@ function civihr_employee_portal_css_alter(&$css) {
 
     _remove_resources($css, $removeFiles);
   } else {
-    _remove_shoreditch_resources($css);
+    if (!strpos(url(current_path(), array('absolute' => TRUE)), 'onboarding-form')) {
```
Comment (mickadoo) [inline civihr_employee_portal/civihr_employee_portal.module]: Please use the `[]` array syntax

Also `drupal_get_path_alias()` returns `onboarding-form` when I'm on the onboarding-form page. Maybe a simple 

```if (drupal_get_path_alias() === 'onboarding-form')```

would be cleaner?

Comment (bmnepali) [reply civihr_employee_portal/civihr_employee_portal.module]: Thanks @mickadoo that's better. Will do!

**Comment by bmnepali** [comment]
Closing this PR and an alternative solution was used to style date-picker in on-boarding page.

**Pull request closed without merging**